cli.py

```diff
@@ -75,5 +84,13 @@ def parse_arguments() -> Tuple[Any, DictConfig]:
         "config_file": args.config or None,
         "seed": args.seed if args.seed else config.get("seed", 8675309),
     })
+    
+    # Overrides for json objects
+    if args.model:
+        config.update({"model": json.loads(args.model)})
+    if args.finetune:
+        config.update({"finetune": json.loads(args.finetune)})
+    if args.pretrain:
+        config.update({"pretrain": json.loads(args.pretrain)})
```

Comment:
This config overriding will get annoying. Would suggest Hydra, omegaconf merging, or argparse default values. See [this other comment](https://github.com/mukobi/Unadaptable-Foundation-Models/pull/16/commits/36d103577ff5072b0f25294c545d2dff2e0b56de#r1558812665)